src/types/requests.ts

```diff
@@ -1,43 +1,43 @@
 import { TokenAmount } from './tokens';
-import { FeeEntry } from '../api';
+import { FeeEntry, MintRequest } from '../api';
 
-// These custom interfaces are used because API.SignableToken is not yet a union type due to OAS 2.0 spec not supporting `OneOf`
+// These custom request interfaces are used because API.SignableToken is not yet a union type due to OAS 2.0 spec not supporting `OneOf`
 // As well as the fact that SignableToken has an extra `data` object which we would like to flatten
 // SignableToken is replaced with AnyToken
 
+/**
+ * Parameter required to create an Order
+ */
 export interface UnsignedOrderRequest {
```

Comment:
this type can probably moved to `orders.ts`?

Reply:
same applies to all other request types in this file. they can probably belong in their respective workflow files. but not blocking